features/step_definitions/basic_steps.rb

```diff
@@ -46,3 +37,18 @@ def page_path_from(page_name)
 Then("I should not see {string}") do |content|
   expect(page).not_to have_content content
 end
+
+def page_path_from(page_name)
+  case page_name.downcase
+    when 'thaitanic' then
+      restaurant_path(Restaurant.find_by(name: page_name))
+    when 'landing' then
+      root_path
+    when 'sign up' then
+      new_user_registration_path
+    when 'cart' then
+      charges_path()
```

Comment:
Small thing, but the parenthesis after line 50 is not needed I'm pretty sure, other than that everything looks good !

Reply:
Fixed :)